main/updater/index.ts

```diff
@@ -0,0 +1,222 @@
+
+import { shell, powerMonitor } from 'electron'
+import log from 'electron-log'
+import path from 'path'
+
+import store from '../store'
+import windows from '../windows'
+import WorkerProcess from '../worker/process'
+
+export interface VersionUpdate {
+  availableUpdate?: {
+    version: string
+    location: string
+  }
+}
+
+function isMac () {
```

Comment:
these funcs can be inlined seeing as you're calling them immediately below e.g. 
`const isMac = process.platform === 'darwin'`